cf-custom-resources/lib/dns-cert-validator.js

```diff
@@ -161,19 +174,100 @@ const requestCertificate = async function (
   return reqCertResponse.CertificateArn;
 };
 
+const updateHostedZoneRecords = async function (
+  action,
+  options,
+  envName,
+  appName,
+  domainName,
+  envRoute53,
+  appRoute53,
+  envHostedZoneId
+) {
+  const promises = [];
+  for (const option of options) {
+    switch (option.DomainName) {
+      case `${envName}.${appName}.${domainName}`:
+        promises.push(validateDomain(
+          envRoute53,
+          option.ResourceRecord,
+          action,
+          "",
```

Comment:
Why do we leave the domain name empty for this scenario?

Reply:
Oh nvm I see, it's either we have to find the `hostedZoneID` or use an existing one.